bot_core.js

```diff
@@ -57,37 +55,71 @@ const handleStart = async (context) => {
 
     sendPreview();
 };
-
-const handleDocument = async (context) => {
+const choose = async (context) => {
     const fileName = context.message.document.file_name;
-
     if (fileName && fileName.endsWith(`.attheme`)) {
-        const theme = await context.downloadFile();
-        const preview = await render({
+        context.reply(
+            'Select the style',{
+                reply_markup:{
+                    inline_keyboard:[[{
+                        text:"Ordinary",
+                        callback_data:"ordinary",
+                        hide:false
+                    },{
+                        text:"Minimalistic",
+                        callback_data:"minimalistic",
+                        hide:false
+                    }]]
+                }, 
+                reply_to_message_id: context.message.message_id
+            }
+            )
+    }
+}
+
+const handleDocument = async (context) => {
+    context.deleteMessage(context.update.callback_query.message.message_id)
+    bot.telegram.sendChatAction(context.update.callback_query.message.chat.id, 'upload_photo')
+    const fileName = context.update.callback_query.message.reply_to_message.document.file_name;
+    const theme = await context.downloadFile();
+    const sendPreview = async (preview) => {
```

Comment:
you don't need to create this function, this code can be run after the following `if` block (almost) as is.